tessera-core/src/main/java/com/quorum/tessera/api/model/SendSignedRequest.java

```diff
@@ -0,0 +1,42 @@
+package com.quorum.tessera.api.model;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import javax.xml.bind.annotation.XmlMimeType;
+
+/**
+ * Model representation of a JSON body on incoming HTTP requests
+ *
+ * Used when a new transaction is to be created where this node is the sender
+ */
+@ApiModel
+public class SendSignedRequest {
+
+    @XmlMimeType("base64Binary")
+    @Size(min = 1)
+    @NotNull
+    @ApiModelProperty("Encrypted payload to send to other parties.")
```

Comment:
Is this a hash or a payload? Assuming its a hash of the payload, in which case the @ApiModelProperty string should be updated.

Reply:
Additionally, assuming it is a hash, then the @Size is fixed and can be updated accordingly. I believe it should always be 64 bytes, but you should double check.

Reply:
Fair enough. I think the same applies there but we should update these together, which can be done later.